etc/config/hooks/live/002-install-pacstall.chroot

```diff
@@ -31,19 +32,31 @@ firmwareType:
     bios:
         -    command: sed -i 's/ cdrom/ [trusted=yes] cdrom/g' /etc/apt/sources.list
         -    timeout: 10
-        -    command: apt-get update --allow-releaseinfo-change
+        -    command: sed -i '/deb http/d' /etc/apt/sources.list
+             timeout: 10
+        -    command: apt-get update
              timeout: 120
-        -    command: apt install -y --no-upgrade -o Acquire::gpgv::Options::=--ignore-time-conflict grub-pc
-             timeout: 300
-        -    command: apt install -y --no-upgrade -o Acquire::gpgv::Options::=--ignore-time-conflict shim-signed
-             timeout: 300'
     "*":
         -    command: sed -i 's/ cdrom/ [trusted=yes] cdrom/g' /etc/apt/sources.list
-        -    timeout: 10
-        -    command: apt-get update --allow-releaseinfo-change
```

Comment:
this is vital for checking if there was a base change (e.g. lunar -> mantic)

Reply:
well the sources are still devel, and the CD is always the same. The user can update after install also considering that there is no internet.